jdat_notebooks/preimaging/preimaging_01_mirage.ipynb

```diff
@@ -0,0 +1,515 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from glob import glob\n",
+    "import shutil\n",
+    "import yaml\n",
+    "\n",
+    "# mirage imports\n",
+    "from mirage import imaging_simulator\n",
+    "from mirage.seed_image import catalog_seed_image\n",
+    "from mirage.dark import dark_prep\n",
+    "from mirage.ramp_generator import obs_generator\n",
+    "from mirage.yaml import yaml_generator\n",
+    "\n",
+    "from astropy.table import Table\n",
+    "from astropy.io import fits\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating simulated images of the JWST astrometric calibration field in the LMC\n",
+    "\n",
+    "\n",
+    "In this tutorial we show step by step instructions to simulate images of the JWST LMC astrometric calibration field. The NIRCam images are simulated using the software MIRAGE. The observation is designed in APT. The APT output is used as input of MIRAGE.\n",
+    "\n",
+    "This Notebook must be executed from an environment that has MIRAGE installed. Follow the instructions in the [Installing MIRAGE webpage](https://mirage-data-simulator.readthedocs.io/en/latest/install.html) before executing this Jupyter Notebook. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting things up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After activating the environment with MIRAGE and beginning a Jupyter Notebook session, we begin by defining the working directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path='/Users/myname/mypath'  # write here your working directory\n",
+    "\n",
+    "os.chdir(path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pwd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Mirage is accompanied by a set of reference files that are used to construct the simulated data. Here we define the location of the MIRAGE data. This is the directory that contains the reference files associated with MIRAGE. \n",
+    "For users at STScI, this is the location of MIRAGE data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['MIRAGE_DATA'] = '/ifs/jwst/wit/mirage_data/'  # this is the STScI location of MIRAGE data"
```

Comment:
I think it's frowned upon to have a full path from the central store in the notebook. Might be best to set this to a dummy value, and make a note that for those on the STScI network, they can email me to ask for the path, as written here: https://mirage-data-simulator.readthedocs.io/en/latest/reference_files.html